lib/spack/spack/test/test_input/CMakeCache.txt

```diff
@@ -0,0 +1,447 @@
+# This is the CMakeCache file.
```

Comment:
I'm all for having decent tests cases, but can this file be shorter and still be useful for tests?  Seems like a minimal test file would be just fine (i.e. header + some of the cache vars)